src/components/Quote.test.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable import/no-unresolved */
+/* eslint-disable no-unused-vars */
+import renderer from 'react-test-renderer';
```

Comment:
- [ ] Test cases are still failing. Kindly ensure to improve upon this aspect.

**Failing test case:**
![image](https://user-images.githubusercontent.com/12745474/141392633-01611864-66d1-47ab-8d9e-3b927399b065.png)

Reply:
Finally fixed